python/tk_hiero_export/sg_audio_export.py

```diff
@@ -184,8 +188,33 @@ def _baseTaskStep(self, item):
                         (item.timelineOut() + handles) + 1,
                     )
 
-                    # If trackitem write out just the audio within the cut
-                    self._sequence.writeAudioToFile(self._audioFile, start, end)
+                    # Versions of Nuke >= 13  require the additional parameters 3 and 2 to be passed to
+                    # the writeAudioToFile method.
+                    # Check Nuke´s version
+                    import nuke
```

Comment:
Maybe this import could be located at the top section?

Reply:
Done!